src/config/knexfile.js

```diff
@@ -6,7 +6,7 @@ const {DB_CLIENT, DB_CONNECTION} = process.env
 
 const options = {
   client: DB_CLIENT || 'sqlite3',
-  connection: DB_CONNECTION || path.join(ROOT, 'data/dev.sqlite3'),
+  connection: DB_CONNECTION || { filename: path.join(ROOT, 'data/dev.sqlite3') },
```

Comment:
thanks for fixing it.

I see now only changes for `dev` env. Could you change it to `test` and `prod` as well to make it consistent?

https://github.com/gothinkster/koa-knex-realworld-example/pull/6/files#diff-97d2396b37935a02e4980ebb4ae41439R32